packages/aws-cdk-lib/aws-elasticloadbalancingv2/test/alb/load-balancer.test.ts

```diff
@@ -264,6 +264,75 @@ describe('tests', () => {
     });
   });
 
+  describe('http2Enabled', () => {
+    test('http2Enabled is not set', () => {
+      // GIVEN
+      const stack = new cdk.Stack();
+      const vpc = new ec2.Vpc(stack, 'Stack');
+
+      // WHEN
+      new elbv2.ApplicationLoadBalancer(stack, 'LB', {
+        vpc,
+      });
+
+      // THEN
+      Template.fromStack(stack).hasResourceProperties('AWS::ElasticLoadBalancingV2::LoadBalancer', {
+        LoadBalancerAttributes: Match.not(
+          Match.arrayWith([
+            {
+              Key: 'routing.http2.enabled',
+              Value: Match.anyValue(),
+            },
+          ]),
+        ),
+      });
+    });
+
+    test('http2Enabled is set to true', () => {
+      // GIVEN
+      const stack = new cdk.Stack();
+      const vpc = new ec2.Vpc(stack, 'Stack');
+
+      // WHEN
+      new elbv2.ApplicationLoadBalancer(stack, 'LB', {
+        vpc,
+        http2Enabled: true,
+      });
+
+      // THEN
+      Template.fromStack(stack).hasResourceProperties('AWS::ElasticLoadBalancingV2::LoadBalancer', {
+        LoadBalancerAttributes: Match.arrayWith([
+          {
+            Key: 'routing.http2.enabled',
+            Value: 'true',
+          },
+        ]),
+      });
+    });
+
+    test('http2Enabled is set to false', () => {
+      // GIVEN
+      const stack = new cdk.Stack();
+      const vpc = new ec2.Vpc(stack, 'Stack');
+
+      // WHEN
+      new elbv2.ApplicationLoadBalancer(stack, 'LB', {
+        vpc,
+        http2Enabled: false,
+      });
+
+      // THEN
+      Template.fromStack(stack).hasResourceProperties('AWS::ElasticLoadBalancingV2::LoadBalancer', {
+        LoadBalancerAttributes: Match.arrayWith([
+          {
+            Key: 'routing.http2.enabled',
+            Value: 'false',
+          },
+        ]),
+      });
+    });
+  });
```

Comment:
How about integrate these 2 tests?

```suggestion
    test.each([true, false])('http2Enabled is set to %s', (http2Enabled) => {
      // GIVEN
      const stack = new cdk.Stack();
      const vpc = new ec2.Vpc(stack, 'Stack');

      // WHEN
      new elbv2.ApplicationLoadBalancer(stack, 'LB', {
        vpc,
        http2Enabled,
      });

      // THEN
      Template.fromStack(stack).hasResourceProperties('AWS::ElasticLoadBalancingV2::LoadBalancer', {
        LoadBalancerAttributes: Match.arrayWith([
          {
            Key: 'routing.http2.enabled',
            Value: String(http2Enabled),
          },
        ]),
      });
    });
```

Reply:
Thanks! I have modified the tests.